createbillboardplaylist.py

```diff
@@ -311,6 +311,14 @@ def create_all(self):
             "http://www.billboard.com/charts/hot-100",
         )
 
+		# Billboard Country 50
+        self.create_playlist_from_chart(
+            "country-songs",
+            "Hot Country",
```

Comment:
Also here, please include the word "Songs" in the title.

```suggestion
            "Hot Country Songs",
```